scenario_player/tasks/execution.py

```diff
@@ -55,10 +55,48 @@ class ParallelTask(SerialTask):
     _name = "parallel"
 
     def _run(self, *args, **kwargs):
-        group = Group()
+        pool = Pool(size=self._config.get("concurrency", None))
```

Comment:
Is there any usecase for a limited pool?

Reply:
It may be useful to limit the concurrency in some cases (for example: when transferring via a hub only do up to X in parallel...)